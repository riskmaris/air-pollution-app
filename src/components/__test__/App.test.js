import React from 'react';
import { render, screen } from '@testing-library/react';
import App from '../../App';
import { Provider } from 'react-redux';
import store from '../../redux/store';
import { BrowserRouter } from 'react-router-dom';

test('renders learn react link', () => {
  const {asFragment} = render( <Provider store={store}>
     <BrowserRouter>
        <App />
      </BrowserRouter>
  </Provider>);
  expect(asFragment()).toMatchSnapshot();
});
