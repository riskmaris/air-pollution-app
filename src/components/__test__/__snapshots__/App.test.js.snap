// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div
      class="header"
    >
      <nav
        class="nav"
      >
        <span
          class="logo"
        >
          <a
            href="/"
          >
            <img
              alt="logo"
              class="back-button"
              src="back.svg"
            />
          </a>
        </span>
        <span
          class="nav-title"
        >
          Crypto Trend
        </span>
        <span
          class="nav-icon"
        >
          <img
            alt=""
            src="microphone.svg"
          />
          <img
            alt=""
            src="setting.svg"
          />
        </span>
      </nav>
    </div>
    <div
      class="container"
    >
      <div
        class="content-container"
        data-testid="home-component"
      >
        <div
          class="loading-container"
          data-testid="loading-container"
        >
          <span />
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
